package.json

```diff
@@ -8,7 +8,7 @@
     "chance": "^1.1.6"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "cd metadata-app && yarn && yarn build && cd .. && rm -r ckanext/dcat_usmetadata/public/**/* && cp -r metadata-app/build/static/* ckanext/dcat_usmetadata/public/ && cd ckanext/dcat_usmetadata/public/ && mv media static/ && mv css/main.*.chunk.css css/main.chunk.css && mv js/2.*.chunk.js js/2.chunk.js && mv js/main.*.chunk.js js/main.chunk.js && mv js/runtime-main.*.js js/runtime-main.js"
```

Comment:
It's great that now we have single command which does all the job. Just one small thing - I would go for splitting this script into 2-3 scripts and reuse them inside the `build` script. Not sure if it makes sense, but first glance this came to my mind.

Reply:
Yes, absolutely @kmanaseryan I'll make small change here.